### DISCUSSION TITLE
Kan man udvikle hold til at kunne dække flere dage?

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#22] [Questions] [answered]
Flere arrangører efterspørger, at  man kan lave hold, der har opgaver på flere forskellige dage. Der er også en ret logisk måde at bruge hold på. 

Lige nu kan man ikke lægge vagter fra forskellige dage ind i samme hold. 

Vil man umiddelbart kunne udvikle det? Eller har det konsekvenser for brugervenlighed, visning, relaterede funktioner mv?

**Comment by mikkelsl** [comment]
Man kan grundlæggende bruge det samme "Hold(navn)", men de vil ikke fremgå under samme dato.
Under siden "Hold" kan man se en oversigt over alle hold, og herunder kan vi senere lave lister over, hvilke events holdet bliver brugt.

**Comment by MartinQwist** [comment]
Men hvis man vælger samme hold på tværs af vagtdage, så benytter de ikke samme chat, og har ikke samme holdleder. Dermed opleves det og er vel reelt også - to forskellige hold. De har bare samme navn - hvilket lige nu kan virke forvirrende. 

Vil det være vanskeligt at udvikle hold til, at hold kan indeholde opgaver på flere vagtdage?

**Reply by mikkelsl** [reply, marked as answer]
Ja, det er ikke tænkt sådan at en chat går på tværs af dage.
Hvis der er et behov for det, har vi brug for bedre at forstå det. Det er vel mere det grupperne er til for.
Man kan alene se det som at "Hold" er en hurtig måde at genbruge et tidligere brugt holdnavn.
